core/deployment/src/main/java/io/quarkus/deployment/pkg/NativeConfig.java

```diff
@@ -243,6 +243,12 @@ public String getEffectiveBuilderImage() {
     @ConfigItem
     public boolean enableVmInspection;
 
+    /**
+     * If JSON build stats should get produced for the native build
```

Comment:
Can we also mention that only works in GraalVM 22.3 + ?

Reply:
Thanks, updated!